File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.js
function solution(nums) {
    var answer = 0;
    
    for(let i=0; i<nums.length-2; i++){
        for(let j=i+1; j<nums.length-1; j++){
            for(let k=j+1; k<nums.length; k++){
                let sum = nums[i]+nums[j]+nums[k];
                
                let check = true;
                for(let cnt=2; cnt <= Math.sqrt(sum); cnt++){
                    if(sum % cnt === 0)
                        check = false;
                }
                
                if(check)
                    answer++;
            }
        }
    }
    
    return answer;
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.cpp
#include <vector>
#include <cmath>
using namespace std;

bool isPrime(int num) {
    for (int i = 2; i <= sqrt(num); i++)
        if (num % i == 0)
            return false;
    return true;
}

int solution(vector<int> nums) {
    int answer = 0;

    vector<int> tmp;
    for(int i=0; i<nums.size()-2; i++){
        for(int j=i+1; j<nums.size()-1; j++){
            for(int k=j+1; k<nums.size(); k++){
                tmp.push_back(nums[i]+nums[j]+nums[k]);
            }
        }
    }
    
    for(int i=0; i<tmp.size(); i++){
        if(isPrime(tmp[i]))
            answer++;
    }

    return answer;
}